src/LibLog/LibLog.cs

```diff
@@ -573,11 +590,14 @@ public bool Log(LogLevel logLevel, Func<string> messageFunc, Exception exception
     internal class LoggerExecutionWrapper : ILog
     {
         private readonly Logger _logger;
+        private readonly Func<bool> _getIsDisabled;
         internal const string FailedToGenerateLogMessage = "Failed to generate log message";
+        private const string EnvironmentVar = "$rootnamespace$_LIBLOG_DISABLE";
```

Comment:
Hoping this will be transformed by VS DTE. Will need to build a package to test it.